src/components/Placeholder/components/PlaceholderContent/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { ReactElement } from 'react';
+import clsx from 'clsx';
+import Typography from '../../../Typography';
+import { Props } from './types';
+
+
+function PlaceholderContent({
+    className,
+    children,
+}: Props): ReactElement {
+
+
+    const placeholderClasses = clsx(
+        'brew-Placeholder--content',
```

Comment:
This className should be `brew-Placeholder__content`.  I described the difference in one of the other review comments.